packages/widgets/src/tabbar.ts

```diff
@@ -893,6 +914,9 @@ class TabBar<T> extends Widget {
     let ci = this._currentIndex;
     let bh = this.insertBehavior;
 
+
+    // TODO: do we need to do an update to update the aria-selected attribute?
```

Comment:
This question seems important for accessibility to actually work as tabs are used. I think answering it is probably within the scope of this pull request in lieu of punting.